server/liveblog/client_modules/client_modules.py

```diff
@@ -36,7 +37,7 @@ class ClientBlogsResource(BlogsResource):
     schema.update(BlogsResource.schema)
 
 
-class ClientBlogsService(BlogService):
+class ClientBlogsService(BaseService):
```

Comment:
please make another commit,
when searching in commit history this will not be clear why it is here.